<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PanelSpecialElements</class>
 <widget class="QWidget" name="PanelSpecialElements">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>609</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_4">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,1">
     <item>
      <widget class="QGroupBox" name="groupBoxCompleteList">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="title">
        <string>Complete list</string>
       </property>
       <layout class="QGridLayout" name="gridLayout_29">
        <item row="0" column="0">
         <widget class="PanelSuperList" name="panelSuperList" native="true">
          <property name="minimumSize">
           <size>
            <width>175</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="gridLayout_3" rowstretch="0,0,0,0,0,0,0,1,1,1,0">
       <item row="1" column="0">
        <widget class="QLabel" name="labelObject">
         <property name="text">
          <string>Object:</string>
         </property>
        </widget>
       </item>
       <item row="4" column="4">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>18</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="9" column="0" colspan="5">
        <widget class="QScrollArea" name="scrollArea">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>457</width>
            <height>68</height>
           </rect>
          </property>
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="0" column="0">
            <widget class="WidgetShowPicture" name="widgetShowPicture" native="true"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item row="8" column="0" colspan="5">
        <widget class="WidgetPreviewObject3D" name="widgetPreviewObject3D" native="true"/>
       </item>
       <item row="4" column="2" colspan="2">
        <widget class="WidgetShape" name="widgetShapeCollisions" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="2">
        <widget class="WidgetShape" name="widgetShapeMtl" native="true">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="labelShape">
         <property name="text">
          <string>Shape:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="2" colspan="3">
        <spacer name="horizontalSpacer_4">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>238</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="labelCollisions">
         <property name="text">
          <string>Collisions:</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelMtl">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>MTL:</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="2">
        <widget class="WidgetPicture" name="widgetPicture" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="labelScale">
         <property name="text">
          <string>Scale:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" colspan="2">
        <widget class="WidgetShape" name="widgetShapeObj" native="true">
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>200</width>
           <height>27</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="comboBoxShape"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="labelTexture">
         <property name="text">
          <string>Texture:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3" rowspan="3" colspan="2">
        <spacer name="horizontalSpacer_5">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>118</width>
           <height>78</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="0" colspan="5">
        <widget class="QGroupBox" name="groupBoxSize">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="title">
          <string>Size</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <layout class="QGridLayout" name="gridLayout">
            <item row="0" column="0">
             <widget class="QLabel" name="labelWidth">
              <property name="text">
               <string>Width:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="QSpinBox" name="spinBoxSquaresWidth"/>
            </item>
            <item row="0" column="2" colspan="2">
             <widget class="QLabel" name="labelWidthSquares">
              <property name="text">
               <string>square(s)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="4">
             <widget class="QLabel" name="label">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item row="0" column="5">
             <widget class="QSpinBox" name="spinBoxPixelsWidth"/>
            </item>
            <item row="0" column="6">
             <widget class="QLabel" name="labelWidthPixels">
              <property name="text">
               <string>pixel(s)</string>
              </property>
             </widget>
            </item>
            <item row="0" column="7" rowspan="3">
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>98</width>
                <height>88</height>
               </size>
              </property>
             </spacer>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="labelHeight">
              <property name="text">
               <string>Height:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QSpinBox" name="spinBoxSquaresHeight"/>
            </item>
            <item row="1" column="2" colspan="2">
             <widget class="QLabel" name="labelHeightSquares">
              <property name="text">
               <string>square(s)</string>
              </property>
             </widget>
            </item>
            <item row="1" column="4">
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item row="1" column="5">
             <widget class="QSpinBox" name="spinBoxPixelsHeight"/>
            </item>
            <item row="1" column="6">
             <widget class="QLabel" name="labelHeightPixels">
              <property name="text">
               <string>pixel(s)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="0">
             <widget class="QLabel" name="labelDepth">
              <property name="text">
               <string>Depth:</string>
              </property>
             </widget>
            </item>
            <item row="2" column="1">
             <widget class="QSpinBox" name="spinBoxSquaresDepth"/>
            </item>
            <item row="2" column="2" colspan="2">
             <widget class="QLabel" name="labelDepthSquares">
              <property name="text">
               <string>square(s)</string>
              </property>
             </widget>
            </item>
            <item row="2" column="4">
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item row="2" column="5">
             <widget class="QSpinBox" name="spinBoxPixelsDepth"/>
            </item>
            <item row="2" column="6">
             <widget class="QLabel" name="labelDepthPixels">
              <property name="text">
               <string>pixel(s)</string>
              </property>
             </widget>
            </item>
            <item row="3" column="0" colspan="8">
             <widget class="Line" name="line">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
             </widget>
            </item>
            <item row="4" column="0">
             <widget class="QLabel" name="labelSizeTexture">
              <property name="text">
               <string>Texture:</string>
              </property>
             </widget>
            </item>
            <item row="4" column="1" colspan="2">
             <widget class="QComboBox" name="comboBoxStretch"/>
            </item>
            <item row="4" column="3" colspan="5">
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>238</width>
                <height>18</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </item>
       <item row="5" column="2" colspan="3">
        <spacer name="horizontalSpacer_6">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>228</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="1">
        <widget class="QDoubleSpinBox" name="doubleSpinBoxScale">
         <property name="decimals">
          <number>4</number>
         </property>
         <property name="value">
          <double>1.000000000000000</double>
         </property>
        </widget>
       </item>
       <item row="7" column="0" colspan="5">
        <widget class="WidgetTilesetSettings" name="widgetTilesetSettings" native="true"/>
       </item>
       <item row="4" column="1">
        <widget class="QComboBox" name="comboBoxCollision"/>
       </item>
       <item row="10" column="0">
        <widget class="QLabel" name="labelCollisionMountains">
         <property name="text">
          <string>Collisions:</string>
         </property>
        </widget>
       </item>
       <item row="10" column="1" colspan="3">
        <widget class="QComboBox" name="comboBoxCollisionMountains"/>
       </item>
       <item row="10" column="4">
        <spacer name="horizontalSpacer_7">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>WidgetPicture</class>
   <extends>QWidget</extends>
   <header>widgetpicture.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetShowPicture</class>
   <extends>QWidget</extends>
   <header>widgetshowpicture.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>PanelSuperList</class>
   <extends>QWidget</extends>
   <header>panelsuperlist.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetPreviewObject3D</class>
   <extends>QWidget</extends>
   <header>widgetpreviewobject3d.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetTilesetSettings</class>
   <extends>QWidget</extends>
   <header>widgettilesetsettings.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>WidgetShape</class>
   <extends>QWidget</extends>
   <header>widgetshape.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
